File: giorno3.java
//fattoriale
import java.util.Scanner;

public class Fattoriale {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);
System.out.print("Inserisci un numero intero positivo: ");
int n = input.nextInt();

long fattoriale = 1;
for (int i = 1; i <= n; i++) {
fattoriale *= i;
}

System.out.println("Il fattoriale di " + n + " è: " + fattoriale);
input.close();
}
}


//somma primi n numeri pari

import java.util.Scanner;

public class SommaNumeriPari {
public static void main(String[] args) {
Scanner input = new Scanner(System.in);

System.out.print("Inserisci un numero: ");
int n = input.nextInt();

int somma = 0;
System.out.println("I primi " + n + " numeri pari sono:");

for (int i = 1; i <= n; i++) {
int pari = 2 * i;
System.out.print(pari + " ");
somma += pari;
}

System.out.println("\nLa somma è: " + somma);
input.close();
}
}

File: giorno3.py
#fattoriale
n = int(input("Inserisci un numero intero positivo: "))

fattoriale = 1
for i in range(1, n + 1):
    fattoriale *= i  

print(f"Il fattoriale di {n} è: {fattoriale}")


#somma primi n numeri pari

n = int(input("Inserisci un numero: "))

somma = 0
print("I primi", n, "numeri pari sono:")

for i in range(1, n + 1):
    pari = 2 * i  # Deve essere indentato
    print(pari, end=" ")
    somma += pari

print("\nLa somma è:", somma)

